.gitignore

```diff
@@ -3,3 +3,4 @@ tmp/
 /npm-debug.log
 .directory
 .kateproject
+.idea/
```

Comment:
I was thinking the same, (either remove kateproject or add idea) as long as we standardise the `/` usage.
Unless a core contributor is using Kate.